lms/templates/api_admin/terms_of_service.html

```diff
@@ -63,8 +63,6 @@ <h2 class="api-subheading">${_("Right to Charge")}</h2>
 
   <p class="api-copy-body">${_("Certain access to the APIs may be provided free of charge, but {platform_name} reserves the right to charge fees for future use or access to the APIs.").format(platform_name=settings.PLATFORM_NAME)}</p>
 
-  <p class="api-copy-body">${_("{platform_name_capitalized} may also update or modify the APIs from time to time without advance notice. These changes may affect your use of the APIs or the way your integration interacts with the API. If we make a change that is unacceptable to you, you should stop using the APIs. Continued use of the APIs means you accept the change.").format(platform_name=settings.PLATFORM_NAME, platform_name_capitalized=settings.PLATFORM_NAME.capitalize())}</p>
```

Comment:
Was this intentionally removed? I didn't see note of it in the JIRA

Reply:
@zrana listed this in the related ticket ENT-988. I have added an explicit note on ENT-982 also.